src/test/java/com/google/jenkins/plugins/computeengine/ComputeEngineCloudIT.java

```diff
@@ -614,7 +613,7 @@ private static void safeDelete(String instanceId, boolean waitForCompletion) {
     }
   }
 
-  private static String logs() {
+  public static String logs() {
```

Comment:
Changing to public?

Reply:
As I said.. Its just the hack before #60